src/index.js

```diff
@@ -1,87 +1,65 @@
 /* eslint-disable prefer-template */
 import './style.css';
 
-let scores = [
-  {
-    name: 'Name: ',
-    score: 127,
-  },
-];
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/X9huMz5c5d7SxuSpxECG/scores/';
 
-const setStored = () => localStorage.setItem('savedScore', JSON.stringify(scores));
-const getStored = () => JSON.parse(window.localStorage.getItem('savedScore'));
-
-class AddScore {
-  constructor(name, score) {
-    this.name = name + ': ';
-    this.score = score;
-  }
+// GET
+async function fetchScore() {
+  const response = await fetch(baseURL);
+  return response.json();
 }
 
-// DOM
-const scoreContainer = document.querySelector('.scores');
-const formContainer = document.querySelector('.form');
-
-// Form
-const ul = document.createElement('ul');
-const liOne = document.createElement('li');
-const inputOne = document.createElement('input');
-const inputTwo = document.createElement('input');
-const liTwo = document.createElement('li');
-const button = document.createElement('button');
-button.classList.add('button');
-const btnClass = document.querySelector('.button');
-const submit = document.createTextNode('Submit');
-button.appendChild(submit);
-formContainer.appendChild(ul);
-ul.appendChild(liOne);
-liOne.appendChild(inputOne);
-ul.appendChild(liTwo);
-liTwo.appendChild(inputTwo);
-formContainer.appendChild(button);
-
-const iterateScore = () => {
-  scores.forEach((item, i) => {
-    scores[i].index = i;
-    const div = document.createElement('div');
-    div.classList.add('container');
-    const name = document.createElement('p');
-    const getName = document.createTextNode(item.name);
-    name.appendChild(getName);
-    const number = document.createElement('p');
-    const getScore = document.createTextNode(item.score);
-    number.appendChild(getScore);
-    div.appendChild(name);
-    div.appendChild(number);
-    scoreContainer.appendChild(div);
+// POST
+// My game ID NVMs7bDqCFiWd9Tmg47Y: console.log(response.json())
+// First we create the ID for the object { name: 'Franks Game'}
```

Comment:
- [ ] Please consider taking off these commented lines as they're not quite relevant and makes your codebase not very presentable and professional. 😕